src/nativeview.ts

```diff
@@ -0,0 +1,14 @@
+import { EventHandler, Failure, GenericCallBack, Success } from "./events";
+
+export interface INativeView {
+    bindAll(events: Map<string, EventHandler>);
+    finish();
+    getValues(fields: string[], success: Success<string>, failure: Failure);
+    hide();
+    render(viewModel: string, success: Success<boolean>, failure: Failure);
+    renderComponent(name: string, model: string, success: Success<boolean>, failure: Failure);
```

Comment:
Method name is failing to tell the difference in it and render() method.

Reply:
chose renderComponent over renderPartial as going forward I visualize ability to export individual component to Calatrava domain.